static/templates/pod.html

```diff
@@ -0,0 +1,12 @@
+<div class="panel panel-default">
```

Comment:
I've updated https://github.com/rapidpro/casepro/pull/110 to be closer to that. You guys can always tweak it later.